test/server/chat-plugins/help.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tests for the Help room auto-answerer.
+ * Written by mia-pi.
+ */
+
+'use strict';
+const assert = require('assert').strict;
+let Help;
+
+describe('Help', function () {
+	before(function () {
+		const defaultData = {
+			stats: {},
+			pairs: {},
+			disabled: false,
+			queue: [],
+		};
+		const Answerer = require('../../../.server-dist/chat-plugins/help').HelpAnswerer;
```

Comment:
why is this not a top level require? you should not need to require this before each test?

Reply:
It requires `Config`, which isn't defined at the top level -- other tests for chatplugins that require `Config` use similar workarounds. (Also, it's not required before *each* test, just before the test set for Help is run.)

Reply:
OK, that's actually quite easy to fix.

Reply:
This workaround will be unnecessary when #7109 is merged.

Reply:
I've just checked: in fact, there aren't any other chat plugins which assume a global ``Config`` exists. So I'd change that...